test/Interpreter/EvalSpec.hs

```diff
@@ -63,6 +64,18 @@ spec = do
 
     testProgramm integers (Integer 4)
 
+    -- chars
+    let chars = [r|
+      let a = 'a'
+      let b = 'b'
+
+      a == b
+      a < b
+      a > b
+      a >= b
+      a <= b
+    |]
```

Comment:
You forgot to do `testProgramm`. In order to test everything at once, you can return a list

```     
[
  a == b,
  a < b,
  a > b,
  a >= b,
  a <= b
]
```

Reply:
Changed, thanks for the idea of testing everything at once through a list. I also added a test for inequality with chars (`a /= b')